### DISCUSSION TITLE
I'm still waiting for the bot DM after following.

### DISCUSSION THREAD

**Discussion by PoorPocketsMcNewHold** [snarfed/bridgy-fed#1019] [Q&A] [answered]
Hi there, I've recently found your service mentionned in a Mastodon toot and wished to experiment it with my personal account.
`@poorpocketsmcnewhold@mamot.fr`

However, after following the bot for ~2 days, I've yet to receive an answer from the bot.
I've tried to "refresh" following, by unfollowing and following back, and by opening up the conversation myself to the bot.
![Screenshot_20240506-085758_1.png](https://github.com/snarfed/bridgy-fed/assets/13170204/7278d376-a54b-4b93-b973-59b675e61632)

I'm hesitating if that was the correct course of action, or if i should have DM'ed the maintainer on their Fediverse account. 
Is they're as well, any public bot status page ?

**Comment by qazmlp** [comment, marked as answer]
Following and the DM flow (once that arrives, it's not here yet I believe) are alternatives for opting in.
You are successfully opted-in and your account is visible at https://bsky.app/profile/poorpocketsmcnewhold.mamot.fr.ap.brid.gy (and the bridge status page for it is at https://fed.brid.gy/ap/@poorpocketsmcnewhold@mamot.fr).

Your posts currently won't federate unless you are followed by at least one **bridged** account (from any other network).  
(The follow right now likely only counts if they follow you *after* opting in to bridging due to #1018.)

@snarfed I think this is another argument for sending opt-in/opt-out confirmation DMs, at least as long as the opt-out wasn't done by blocking.

**Comment by snarfed** [comment]
> @snarfed I think this is another argument for sending opt-in/opt-out confirmation DMs, at least as long as the opt-out wasn't done by blocking.

Agreed! I've filed https://github.com/snarfed/bridgy-fed/issues/1024

**Reply by DoktorZjivago** [reply]
An additional question to this. 

I opted in from mastodon a few days ago, initially everything worked, then suddenly since two days, nothing gets bridged anymore. However I see accounts on the fediverse that more recently opted in (like a day or two) that seems to function.

I also opted in from bluesky, but a fediverse mirror was never created. I thought maybe it was due to the "discourage visibility in logged out view" option, so I switched off that and re-followed but still no mirror account on the fediverse. 

Lots of questions here and I'm well aware the process still is in an early stage, just wanted to share what I saw happened. Every small progress on this bridge is like a Christmas present lol ☺️

**Reply by DoktorZjivago** [reply]
Oh I noticed another thing. It seems accounts on the fediverse that opted in five days or longer ago, does not get bridged to bluesky since the last three/four days. Something happened there.
